org.flatpak.CineEncoder.yaml

```diff
@@ -0,0 +1,24 @@
+
+app-id: org.flatpak.CineEncoder
```

Comment:
The app-id is likely incorrect since this is not developed by Flatpak. It should be either based on the domain used by the project, or the github hosting.